azure/iac/storageaccounts.rego

```diff
@@ -726,3 +726,348 @@ storage_account_latest_tls_configured_metadata := {
     "Policy Help URL": "",
     "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.storage/storageaccounts"
 }
+
+# PR-AZR-ARM-STR-019
+#
+default storage_account_private_endpoint = null
+
+azure_attribute_absence["storage_account_private_endpoint"] {
+    count([c | lower(input.resources[_].type) == "microsoft.storage/storageaccounts"; c := 1]) != count([c | lower(input.resources[_].type) == "microsoft.network/privateendpoints"; c := 1])
+}
+
+azure_issue["storage_account_private_endpoint"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.network/privateendpoints"
+    privateLinkServiceConnection := resource.properties.privateLinkServiceConnections[_]
+    contains(lower(privateLinkServiceConnection.properties.privateLinkServiceId), "microsoft.storage/storageaccounts")
```

Comment:
@shahinshirvani either add a `not `before `contains `or rename `azure_issue `to `no_azure_issue `for readability purpose.